### ISSUE TITLE
multi cards support？

### ISSUE THREAD

**Issue by litt1eXixi** [ToTheBeginning/PuLID#88]
﻿
Now all the weights are loaded into one card. Can we load the weights into multiple cards to avoid out of memory errors

**Comment by ToTheBeginning** [comment]
Yes, you can refer to https://github.com/ToTheBeginning/PuLID/issues/75#issuecomment-2359886134 and hack the codes.
